CrrExistingObjects/metadataClient.js

```diff
@@ -0,0 +1,38 @@
+const { MetadataWrapper } = require('arsenal').storage.metadata;
+const werelogs = require('werelogs');
+const listingParser = require('./listingParser');
+
+const loggerConfig = {
+    level: 'info',
+    dump: 'error',
+};
+werelogs.configure(loggerConfig);
+
+const log = new werelogs.Logger('s3utils::crrExistingObjects');
+const replicaSetHosts = process.env.MONGODB_REPLICASET;
+const replicaSet = 'rs0';
+const writeConcern = 'majority';
+const readPreference = 'primary';
+const database = process.env.MONGODB_DATABASE || 'metadata';
+const implName = 'mongodb';
+const params = {
+    customListingParser: listingParser,
+    mongodb: {
+        replicaSetHosts,
+        writeConcern,
+        replicaSet,
+        readPreference,
+        database,
+    },
+    replicationGroupId: 'RG001',
```

Comment:
this should be configurable, as it can (should) be set differently on a multi-site deployment.

Reply:
Is it only `replicationGroupId`? or others too?

Reply:
I don't know what this variable is used for, but if it relies on it to fetch keys in mongodb, it might fail if the deployment has been configured with a different replication group ID. If the client does not rely on it to fetch keys, you can probably ignore this. (Otherwise, yes, you can make it configurable with an env var).

Reply:
`replicationGroupId` is used for generating versionId's, which is in turn used in creating metadata objects. Since we are creating new objects using the script, I will introduce a new env var.

Reply:
I suggest to make it a mandatory env var, to force the user to consider it and avoid mistakes.